torchvision/prototype/transforms/functional/_augment.py

```diff
@@ -14,3 +17,13 @@ def erase_image_pil(
     t_img = pil_to_tensor(img)
     output = erase_image_tensor(t_img, i=i, j=j, h=h, w=w, v=v, inplace=inplace)
     return to_pil_image(output, mode=img.mode)
+
+
+def erase(inpt: Any, i: int, j: int, h: int, w: int, v: torch.Tensor, inplace: bool = False):
```

Comment:
This dispatcher was missing. We agreed offline that we should always have a dispatcher in case we have more than one kernel. Thus, although this is a image-only function, a dispatcher is still warranted since we have a tensor and PIL kernel.